stories/resizable-card-layout/resizable-card-layout.stories.js

```diff
@@ -0,0 +1,505 @@
+export default {
+    title: 'Layouts/Resizable Card Layout',
+    parameters: {
+        description: `The resizable card layout is a layout for the overview page. It enables users to define a personalized card layout by changing not only the position of a card, but also its size, and thus how the card content is presented.
+        
+This layout gives users much greater flexibility in tailoring the overview page to their specific business needs. And it allows app teams to offer varying levels of detail for any given card. Whenever the size of a card changes, the content adapts automatically to show the most relevant information in the available space.
+
+Cards can be resized vertically in rows of 1 rem and horizontally in steps of 20 rem (minimum width).
+        `,
+        tags: ['f3', 'theme', 'development'],
+        components: [
+            'resizable-card-layout',
+            'button',
+            'avatar',
+            'badge',
+            'card',
+            'object-status',
+            'numeric-content',
+            'table',
+            'checkbox',
+            'list',
+            'link',
+            'icon'
+        ]
+    }
+};
+
+export const resizeCardLayout = () =>
+    `<div id="grid1" class="fd-resizable-card-layout fd-resizable-card-layout--md fd-custom-resizable-card-layout-flex">
+        <div id="one" class="fd-resizable-card-layout__wrapper card-one-dimensions">
+            <div class="fd-resizable-card-layout__wrapper__resize__handle--vertical"></div>
```

Comment:
Could you reduce naming to `fd-resizable-card-layout__resize--vertical`?

Reply:
fixed.